packages/mds-logger/index.ts

```diff
@@ -51,16 +54,14 @@ const log =
     const log_ISO_timestamp = new Date(log_timestamp).toISOString()
     const log_requestId = httpContext.get('x-request-id')
```

Comment:
I wonder if here, we could no something like this to make it clear when the request ID isn't coming through:
```ts
const log_requestId = httpContext.get('x-request-id') ?? 'not-set'
```

Reply:
I'm concerned that could throw a tracing framework for a loop, I'd definitely wanna have a jaeger env (or something similar) spun up in a dev cluster to test that it doesn't create some insanely weird traces that blow OOM.